src/main/java/net/sf/jabref/gui/JabRefFrame.java

```diff
@@ -322,7 +322,6 @@
     private final Action toggleToolbar = enableToggle(new AbstractAction(Localization.menuTitle("Hide/show toolbar")) {
 
         {
-            putValue(Action.ACCELERATOR_KEY, Globals.getKeyPrefs().getKey(KeyBinding.HIDE_SHOW_TOOLBAR));
```

Comment:
@tobiasdiez suggested this change because i quote: 

> Remove default Toggle Toolbar shortcut  (can't imagine that this is useful for somebody, I would leave alt + num for toggling real panes)